spa/src/oauth-manager/index.tsx

```diff
@@ -37,7 +37,7 @@ const OauthManager = () => {
 		if (state !== prevState) return false;
 
 		const token = await ApiRequest.githubAuth.exchangeToken(code, state);
-		if (token.data.accessToken && token.data.refreshToken) {
+		if (token.data.accessToken) {
```

Comment:
On stage our app is not configured to return refresh token. Hence it will always be empty.